printer_mods/Kyleisah/CableGator/README.md

```diff
@@ -0,0 +1,75 @@
+# The CableGator!
+The cable comb that takes a BITE out of dragchain frustration!
+
+<img src="Images/StylishView.png" width="480px" />
+
+## Why do I need this?
+Keeping your wiring tidy as you're installing your dragchains can be a hassle, sometimes quite the croc!
+
+Enter, CableGator!
+
+The totally not reptilian overlord that gently combs your "I swear it was neat but now it's a rat's nest" wiring harness into a tame and beautiful bundle of joy, ready for electrocution.
+
+## Aren't gators dangerous?
+Yes!
+
+## How do I make this?
+
+CableGator was designed and prototyped by a stranger you've never met, on the internet. But that stranger also mentioned something about using the typical Voron Parts slicing configuration (forced .4 extrusion widths and .20mm layer height) to print this. It has been designed with 3D printing in mind! The CableGator can be made with all kinds of different 3D printing materials, but molten plastic is recommended.
+
+## What all do I need?
+
+#### Materials:
+#### - (1) M3 Heat-Set insert
+#### - (1) M3x16 SHCS
+#### - (Both) CableGator [parts](https://github.com/kyleisah/VoronUsers/tree/master/printer_mods/Kyleisah/STLs), fresh off the print bed.
```

Comment:
can you make this a relative link?

    [parts](./STLs)

Reply:
Whoops! Consider it done, just pushed the new commit.